cellxgene_schema_cli/scripts/gene_processing.py

```diff
@@ -84,14 +84,19 @@ def _parse_gtf(self, gtf_path: str, gene_info_description: str):
 
                 # Extract features (column 9 of GTF)
                 current_features = gtf_tools._get_features(line)  # type: ignore
+
+                # Set gene_name to gene_id if not present in GTF line
```

Comment:
jason confirmed we can just set the gene name to be the gene id: https://czi-sci.slack.com/archives/C02TSE92DBM/p1729717238997349